README.md

```diff
@@ -127,18 +127,18 @@ When running the Release Manager, default configuration values are used. The
 most relevant values are:
 
 - `-healthcheck-offset=30m`: When assessing the candidate's health, the Release
```

Comment:
it's almost like we should document these bullet points right after deploying with these params.

Reply:
We could add a link to the section "Default rollout configuration"